crates/cli-support/src/js/mod.rs

```diff
@@ -500,6 +525,32 @@ impl<'a> Context<'a> {
                 footer.push_str("export { initSync };\n");
                 footer.push_str("export default __wbg_init;");
             }
+
+            OutputMode::Isomorphic => {
+                js.push_str(&self.generate_node_imports());
+
+                js.push_str("let wasm;\n");
+
+                for (id, js) in crate::sorted_iter(&self.wasm_import_definitions) {
+                    let import = self.module.imports.get_mut(*id);
+                    footer.push_str("\nmodule.exports.");
```

Comment:
The use-cases all expect CommonJS?

Reply:
Good point. Technically it also makes sense for ecmascript modules but I can't think of a use case.
[ ] check whether and how to support ecmascript modules

Reply:
I decided not to use ecmascript modules for now as I don't see a use case for this. The right way of mixing ecmascript modules with wasm should be importing wasm as esm.